gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -159,6 +159,10 @@
         </div>
       </div>
     </form>
+    <div class="form-group" ng-if="willBeReintroducedBy">
+      <label translate for="removal">Device Removal</label>
+      <p class="help-block" translate translate-value-introducer="{{ willBeReintroducedBy }}">When you remove this device, it might be automatically reintroduced by {%introducer%}.</p>
+    </div>
```

Comment:
I didn't properly explain what I meant in my last comment: I intended to remove/replace this part with the notice in the remove device dialog, i.e. to remove this div block.

Reply:
Sorry it was clear, I just went too fast and should have double-checked.